web-console/console-config.js

```diff
@@ -19,4 +19,9 @@
 window.consoleConfig = {
   exampleManifestsUrl: 'https://druid.apache.org/data/example-manifests-v2.tsv',
   /* future configs may go here */
+  defaultQueryContext: {
```

Comment:
Is this also a change that is needed for the linter to pass?

Reply:
Updated